src/suggestionlistworker.cpp

```diff
@@ -8,6 +9,12 @@ SuggestionListWorker::SuggestionListWorker(const QString& text, int token, QObje
 {
 }
 
+std::string lcAll(std::string title)
```

Comment:
We can remove this too, not used anywhere now

Reply:
Removed. Thanks